config/load.go

```diff
@@ -45,13 +45,24 @@ func Load(logger *zerolog.Logger) (appConfig *AppConfig, err error) {
 		if err != nil {
 			logger.Error().Msg("Unable to decode App Config to json")
 		} else {
-			fmt.Printf("loaded config: %s", cfg)
+			logger.Debug().Msgf("loaded config: %s", cfg)
 		}
 	}
 
 	return appConfig, nil
 }
 
+func LoadForTest() (appConfig *AppConfig) {
```

Comment:
Before merging - fix this lint error: `exported function LoadForTest should have comment or be unexported (revive)`

Reply:
fixed